centos_ci.sh

```diff
@@ -124,14 +124,14 @@ function upload_logs() {
 }
 
 function run_tests() {
-  set +e
-  # In Jenkins slave we have pull secret file in the $HOME/payload/crc_pull_secret
-  # this is copied over using https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml#L99
-  make integration BUNDLE_LOCATION=$HOME/Downloads/$BUNDLE PULL_SECRET_FILE=$HOME/payload/crc_pull_secret
-  if [[ $? -ne 0 ]]; then
-    upload_logs $1
-    exit 1
-  fi
+	set +e
+	# In Jenkins slave we have pull secret file in the $HOME/payload/crc_pull_secret
+	# this is copied over using https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml#L99
+	make integration BUNDLE_LOCATION=$HOME/Downloads/$BUNDLE PULL_SECRET_FILE=$HOME/payload/crc_pull_secret
+	if [[ $? -ne 0 ]]; then
+		upload_logs $1
+		exit 1
+	fi
```

Comment:
@jsliacan  This changes shouldn't be part of this PR. it is just matter of test.